client/my-sites/hosting/github/deployment-card/style.scss

```diff
@@ -0,0 +1,16 @@
+.deployment-card__row {
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+	gap: 48px;
+	margin-bottom: 8px;
+}
+
+.deployment-card__column-title {
+	font-weight: 600;
+}
```

Comment:
This class is now a leftover. Feel free to remove it, or reuse it by replacing the `<strong>` element in the component.

One thing we could to is to move the inline CSS rules from the component to a class name.

Reply:
we already using a good amount of inline styles, Let's keep it that way for now